.gitignore

```diff
@@ -0,0 +1,4 @@
+./bin/
+capture.pcap
+tmp/
+*.exe
```

Comment:
I'd hope exe files are all in `/bin/`…

Reply:
make places them there, but incase if someone builds not using make.